src/etc/inc/easyrule.inc

```diff
@@ -292,7 +292,7 @@ function easyrule_pass_rule_add($int, $proto, $srchost, $dsthost, $dstport, $ipp
 	$filterent['type'] = 'pass';
 	$filterent['interface'] = $int;
 	$filterent['ipprotocol'] = $ipproto;
-	$filterent['descr'] = gettext("Easy Rule: Passed from Firewall Log View");
+	$filterent['descr'] = gettext($desc);
```

Comment:
OK, then would passing variable directly to $filterent['descr'] be ok for you? This would be needed to call the easyrule script using ansible (or any other similar automation tools) to dynamically configure rules